test/integration/resourcemanager/garbagecollector/garbagecollector_suite_test.go

```diff
@@ -123,7 +123,7 @@ var _ = BeforeSuite(func() {
 		Clock:                 clock.RealClock{},
 		MinimumObjectLifetime: pointer.Duration(0),
 		// Use the same version as the envtest package
-		TargetKubernetesVersion: semver.MustParse("1.27.0"),
+		TargetKubernetesVersion: semver.MustParse("1.28.0"),
```

Comment:
There is also https://github.com/gardener/gardener/blob/b60abf22a5a7d5d1382fdab4b2fba80372f0b9a2/test/integration/resourcemanager/highavailabilityconfig/highavailabilityconfig_suite_test.go#L123-L124

Reply:
If https://github.com/gardener/gardener/pull/8487 gets merged before, then we're good.

Reply:
Seems like the case.